### DISCUSSION TITLE
Issue with video thumbnail

### DISCUSSION THREAD

**Discussion by imsarkyhiggs** [pimcore/pimcore#12549] [Q&A / Support]
When clicking on a video, specifically in the DAM section of pimcore, I don't get a thumbnail/preview. Instead, i get an error:

`Pimcore\Model\Asset\Video\ImageThumbnail::__toString(): Return value must be of type string, null returned`

As far as I can tell, the error is in code that I haven't touched. For reference, I'm running this on ROCKY 8 through apache (httpd). I've attached a picture of it as well. Anyone have any ideas on what is happening or how i can fix this? 

Thanks!

![image](https://user-images.githubusercontent.com/108363924/176256522-887cdc70-09dd-4160-af0c-69b422b7e5a9.png)
![image](https://user-images.githubusercontent.com/108363924/176256597-6ddb9916-ffc4-4faa-b8fd-f47f7155bb6d.png)

**Comment by dvesh3** [comment]
this might help you https://github.com/pimcore/pimcore/discussions/11876.

The preview is generated by Symfony messenger, so please make sure you have maintenance setup properly.

**Comment by blankse** [comment]
@imsarkyhiggs @dvesh3 Maybe related: https://github.com/pimcore/pimcore/pull/12697
